dll/win32/framedyn/chstring.cpp

```diff
@@ -1004,7 +1004,7 @@ int CHString::ReverseFind(CHSTRING_WCHAR ch) const
     CHSTRING_WCHAR *Last;
 
     // Let's use appropriate helper
-    Last = reinterpret_cast<CHSTRING_WCHAR*>(wcsrchr(reinterpret_cast<LPCWSTR>(m_pchData), ch));
+    Last = reinterpret_cast<CHSTRING_WCHAR*>(wcsrchr(reinterpret_cast<LPWSTR>(m_pchData), ch));
```

Comment:
Wouldn't it be better to cast the result to `const CHSTRING_WCHAR*` instead?

Reply:
It doesn't make much of difference, but since it doesn't require any additional casts or anything, why not.